client/packages/system/src/Name/ListView/ImportProperties/PropertiesImportModal.tsx

```diff
@@ -0,0 +1,239 @@
+          setImportProgress(percentComplete);
+          setImportErrorCount(importErrorRows.length);
+        });
+      }
+      if (importErrorRows.length === 0) {
+        const importMessage = t('messages.import-generic', {
+          count: numberImportRecords,
+        });
+        const successSnack = success(importMessage);
+        successSnack();
+        onChangeTab(Tabs.Upload);
+        setBufferedFacilityProperties([]);
+        setErrorMessage('');
+        onClose();
+      } else {
+        // Load the error rows in to the component for review
+        setErrorMessage(t('messages.import-generic'));
+        setBufferedFacilityProperties(importErrorRows);
+        setImportErrorCount(importErrorRows.length);
+        onChangeTab(Tabs.Review);
+      }
+    }
+  };
+
+  const onClickStep = (tabName: string) => {
+    switch (tabName) {
+      case Tabs.Upload:
+        changeTab(tabName as Tabs);
+        break;
+      case Tabs.Review:
+        changeTab(tabName as Tabs);
+        break;
+      case Tabs.Import:
+        // Do nothing, user can't get to the import page without clicking the import button
+        break;
+    }
+  };
+
+  const changeTab = (tabName: Tabs) => {
+    switch (tabName) {
+      case Tabs.Upload:
+        setErrorMessage('');
+        setActiveStep(0);
+        break;
+      case Tabs.Review:
+        setActiveStep(1);
+        break;
+      case Tabs.Import:
+        setActiveStep(2);
+        break;
+    }
+    onChangeTab(tabName);
+  };
+
+  const showWarnings = errorMessage.length == 0 && warningMessage.length > 0;
+
+  console.info('showWarnings', showWarnings);
+
+  const importSteps = [
+    { label: t('label.upload'), description: '', clickable: true },
+    { label: t('label.review'), description: '', clickable: true },
+    {
+      label: t('label.import'),
+      description: '',
+      clickable: false,
+    },
+  ];
+
+  return (
+    <Modal
+      okButton={
+        <DialogButton variant="next" disabled={false} onClick={importAction} />
+      }
+      cancelButton={
+        <DialogButton
+          variant="cancel"
+          onClick={async () => {
+            setErrorMessage('');
+            changeTab(Tabs.Upload);
+            onClose();
+          }}
+        />
+      }
+      nextButton={
+        <DialogButton variant="export" disabled={false} onClick={() => {}} />
+      }
+      title={t('label.import-cce')}
+      height={1000}
+      width={1600}
+    >
+      <>
+        <ClickableStepper
+          steps={importSteps}
+          activeStep={activeStep}
+          onClickStep={onClickStep}
+        ></ClickableStepper>
+        {errorMessage ? <Alert severity="error">{errorMessage}</Alert> : null}
+        <TabContext value={currentTab}>
+          <Grid container flex={1} flexDirection="column" gap={1}>
+            <Grid item display="flex">
+              <Box flex={1} flexBasis="40%"></Box>
+              <Box flex={1} flexBasis="60%"></Box>
+            </Grid>
```

Comment:
What are these empty boxes doing?

Reply:
!!! they've been carried around throughout all of our import modals